cve_bin_tool/error_handler.py

```diff
@@ -0,0 +1,131 @@
+import sys
+from enum import Enum
+
+from rich.console import Console
+from rich.traceback import Traceback
+
+CONSOLE = Console()
+
+
+class InsufficientArgs(Exception):
+    """ Insufficient command line arguments"""
+
+
+class InvalidCsvError(Exception):
+    """ Given File is an Invalid CSV """
+
```

Comment:
Moved all exception to one place in error_handler so that new contributor can easily see which exit code is used and always give unique exit code.